README.md

```diff
@@ -1,45 +1,81 @@
+
+# Anatomy of a shield study add-on
+
+Shield study add-ons are legacy add-ons with an embedded webextension. It needs to be a legacy add-on in order to be able to access Telemetry data, user preferences etc that are required for collecting relevant data for [Shield Studies](https://wiki.mozilla.org/Firefox/Shield/Shield_Studies).
 
 # Functionality
 
 This add-on assigns users to `test` and `control` groups at random. The `test` group will receive a pop-up after successfully loading a URI. The pop-up has a "Click Here!" button, and a telemetry ping is sent if the button is clicked. The pop-up should collapse itself after clicking anywhere outside the pop-up and never be displayed again. 
 
 # Getting Started
-After cloning the repo, you can run the following commands from the top level directory:
+After cloning the repo, you can run the following commands from the top level directory, one after another:
 
-`$ npm install`
+`$ cd shield-integrated-addon/ && npm install`
 
-`$ cd shield-study-embedded-webextension-hello-world-example/shield-integrated-addon/addons/example/ && npm run build`
+`$ cd addons/example/ && npm run build`
 
-This packages the add-on into  `addon.xpi` which is stored in `/shield-integrated-addon/addons/example/dist/`. This file is what you load into Firefox. 
+This packages the add-on into `addon.xpi` which is stored in `/shield-integrated-addon/addons/example/dist/`. This file is what you load into Firefox. 
 
 # Loading
+
 Once you package your `.xpi` file, open (preferably) the [Developer Edition of Firefox](https://www.mozilla.org/firefox/developer/). You can load the `.xpi` using the following steps: 
 
 * Navigate to *about:config* and set `extensions.legacy.enabled` to `true`. This permits the loading of the embedded WebExtension since new versions of Firefox are becoming restricted to pure WebExtensions only. 
 * Navigate to *about:debugging* in your URL bar
 * Select "Load Temporary Add-on"
 * Find and select the `.xpi` file you just built.
 
+# Seeing the add-on in action
+
+I recommend using the Browser Console which can be open from Firefox's top toolbar in Tools > Web Developer > Browser Console. This will display Shield (loading/telemetry) and `console.log()` output.
+
+To see the Add-on at work, navigate to any url (for instance [http://example.com]()) and you should see a green puzzle piece icon in the browser address bar. You should also see the following in the Browser Console, which comes from this add-on:
+
+```
+Heard a web nav...  background.js:55:13
+```
+
+If it's the first web navigation you do, you'll also notice a popup and the following log messages:
+
+```
+triggering popup  bootstrap.js:69
+```
+
+You can also manually trigger the popup by clicking the green puzzle icon in the address bar, whereupon you should see the following in the Browser Console:
```

Comment:
Clicking on the green puzzle piece does not trigger the output below. This output is generated when a user clicks on the "Click Here!" button within the pop-up.

Reply:
Yep, my bad, pushed correction, thanks